### DISCUSSION TITLE
Longfast messages IN on MQTT but missing on APP/WEB

### DISCUSSION THREAD

**Discussion by MrSiO** [meshtastic/firmware#4983] [Troubleshooting]
### Category

Other

### Hardware

Heltec V3

### Firmware Version

2.4.3.91

### Description

I have a node with LongFast channel connected to my private MQTT broker on my LAN trough wifi.

Alot of messages from nearby nodes are coming in on LongFast and are forwarded to my MQTT broker (I can see messages from my MQTT client connected to same topic) but most messages (not all) are not coming into my Android app nor my web interface using client.meshtastic.og.

How is that possible?

Lora connection has to be OK since I do get messages and they are forwarded to MQTT but no notification otherwise.. If I didn't have my node connected to broker I would have known messages were coming in...

Any ideas??

Note: This node is configured as Client_Mute because it's in a basement and I have a roof node as Client with 8dbi antenna for range. So basically all my inbound and outbound communications are hopping trough my roof node. 

### Relevant log output

```Shell
Nothing to show for the moment since no messages are coming in...
```

**Comment by fifieldt** [comment]
There are per-channel settings for downlinking and uplinking, did you check those?
